### PR TITLE
WASM: Enable System.Reflection.* tests

### PR DESCRIPTION
**Pull request by akoeplinger**
Disable tests that won't work on browser.

### CODE REVIEW

File: eng/testing/tests.mobile.targets [hunk 1]
```diff
@@ -147,11 +147,19 @@
...
     <ItemGroup Condition="'$(AssemblyName)' == 'System.Reflection.MetadataLoadContext.Tests'">
       <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Private.CoreLib.dll" />
       <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Reflection.MetadataLoadContext.Tests.dll" />
       <WasmFilesToIncludeInFileSystem Include="$(PublishDir)mscorlib.dll" />
     </ItemGroup>
+    <ItemGroup Condition="'$(AssemblyName)' == 'System.Reflection.Metadata.Tests'">
+      <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Reflection.Metadata.Tests.dll" />
+    </ItemGroup>
+    <ItemGroup Condition="'$(AssemblyName)' == 'System.Reflection.Tests'">
+      <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Reflection.Tests.dll" />
```
Comment (ViktorHofer) [inline eng/testing/tests.mobile.targets:160]: is there a better way to track these dependencies? hardcoding looks wrong

**Review by ViktorHofer — APPROVED** [review]

File: eng/testing/tests.mobile.targets (hunk 1 above)
Comment (ViktorHofer) [reply eng/testing/tests.mobile.targets:160]: Or avoid defining this centrally and instead set a property in the project file that indicates this?

Comment (akoeplinger) [reply eng/testing/tests.mobile.targets:160]: yeah we need to find a better way.

Comment (ViktorHofer) [reply eng/testing/tests.mobile.targets:160]: Can you please open an issue to discuss this?

Comment (safern) [reply eng/testing/tests.mobile.targets:160]: I think the project could add to an item and then we could transform that item to remap the value to be under `PublishDir`

**Pull request merged**